.github/workflows/release.yml

```diff
@@ -2,8 +2,10 @@ name: "Release"
 
 on:
   push:
+    branches:
+      - "RC[0-9]/**"
     tags:
-      - "v[0-9]+.[0-9]+.[0-9]+" # Push events to matching v*, i.e. v1.0, v20.15.10
+      - "v[0-9]+.[0-9]+.[0-9]+"  # Push events to matching v*, i.e. v1.0, v20.15.10
```

Comment:
This is copied as-is from `master`.

Why is it that we don't cut RCs from tags? Just asking for context.

Reply:
We generally _have_ cut release candidates using tags (not separate backport branches). Or are you asking why we don't match release candidates for invocations of the binary release builder?

Reply:
I suppose I was just wondering why this workflow's executed when pushing to the `RC[0-9]/**` branches?

Reply:
I suspect that's historical baggage. I don't think we've used explicit RC branches in recent memory.

Reply:
Good to know, I may just take it out.